src/main/java/com/theelixrlabs/healthcare/constants/MessageConstants.java

```diff
@@ -5,20 +5,25 @@
  * These keys are mapped to actual messages in the properties file.
  */
 public class MessageConstants {
-    public static final String AADHAAR_ALREADY_PRESENT = "doctor.aadhaarNumber.already.present";
-    public static final String AADHAAR_NUMBER_SHOULD_BE_MANDATORY = "{doctor.aadhaarNumber.mandatory}";
-    public static final String AADHAAR_SHOULD_NOT_BE_EMPTY = "doctor.aadhaarNumber.should.not.be.empty";
     public static final String DEPARTMENT_SHOULD_BE_MANDATORY = "{doctor.department.mandatory}";
-    public static final String DOCTOR_DEPARTMENT_SHOULD_NOT_BE_EMPTY = "doctor.department.should.not.be.empty";
-    public static final String DOCTOR_ID_IS_MANDATORY = "{validation.doctorID.mandatory}";
+    public static final String DEPARTMENT_SHOULD_NOT_BE_EMPTY = "{doctor.department.should.not.be.empty}";
+    public static final String DOCTOR_AADHAAR_ALREADY_PRESENT = "doctor.aadhaar.number.already.present";
+    public static final String DOCTOR_AADHAAR_NUMBER_SHOULD_BE_MANDATORY = "{doctor.aadhaar.number.mandatory}";
+    public static final String DOCTOR_AADHAAR_NUMBER_SHOULD_NOT_BE_EMPTY = "doctor.aadhaar.number.should.not.be.empty";
+    public static final String DOCTOR_FIRST_NAME_SHOULD_BE_MANDATORY = "{doctor.first.name.mandatory}";
+    public static final String DOCTOR_FIRST_NAME_SHOULD_NOT_BE_EMPTY = "doctor.first.name.should.not.be.empty";
+    public static final String DOCTOR_ID_IS_MANDATORY = "validation.doctor.id.mandatory";
+    public static final String DOCTOR_INVALID_AADHAAR_NUMBER = "doctor.invalid.aadhaar.number";
+    public static final String DOCTOR_INVALID_FIRSTNAME = "doctor.invalid.first.name";
+    public static final String DOCTOR_INVALID_LASTNAME = "doctor.invalid.last.name";
+    public static final String DOCTOR_LAST_NAME_SHOULD_BE_MANDATORY = "{doctor.last.name.mandatory}";
+    public static final String DOCTOR_LAST_NAME_SHOULD_NOT_EMPTY = "doctor.last.name.should.not.be.empty";
```

Comment:
rename DOCTOR_LAST_NAME_SHOULD_NOT_EMPTY to DOCTOR_LAST_NAME_SHOULD_NOT_BE_EMPTY

Reply:
Changed .